src/renderer/dx/DxRenderer.cpp

```diff
@@ -1345,6 +1346,9 @@ try
     RETURN_IF_FAILED(_customLayout->Reset());
     RETURN_IF_FAILED(_customLayout->AppendClusters(clusters));
 
+    // Copy cursor info into drawing context

```

Comment:
BTW This describes the "what" and not the "why", the former of which is already written down as code. 🙈

Reply:
yeah it's gone and I just stuff it straight where it needs to go instead of multiphase it. thanks.